ofrak_core/ofrak/core/apk.py

```diff
@@ -81,15 +82,18 @@ async def unpack(self, resource: Resource, config=None):
             temp_file.write(data)
             temp_file.flush()
             with tempfile.TemporaryDirectory() as temp_flush_dir:
-                command = [
+                proc = await asyncio.create_subprocess_exec(
                     "apktool",
                     "decode",
                     "--output",
                     temp_flush_dir,
                     "--force",
                     temp_file.name,
-                ]
-                subprocess.run(command, check=True, capture_output=True)
+                    stderr=asyncio.subprocess.PIPE,
+                )
+                stdout, stderr = await proc.communicate()
+                if proc.returncode and proc.returncode < 0:
```

Comment:
According to [the `asyncio` docs](https://docs.python.org/3/library/asyncio-subprocess.html#asyncio.subprocess.Process.returncode), the return code is only negative if a signal was sent to terminate the process. If the process returns any other nonzero exit status, this will not catch it.

The following suggestion captures checking if `proc.returncode is not None` as well as checking that `proc.returncode != 0` by combining both of those checks into one. 

```suggestion
                if proc.returncode:
```

This comment applies to all of the places throughout this PR where changes to `asyncio.create_subprocess_exec` were made.